feat: prompt user to select device type (#195)
salesforcecli/plugin-lightning-dev: messages/prompts.md

# site.select

Select a site

# site.confirm-update

> An updated site bundle is available for "%s". Would you like to download and apply the update?

Reviewer: just a suggestion!  But I think "do you want to" is more direct than "would you like to", and direct is good, IMHO.

# device-type.title

> Which device type would you like to use for preview

Author: In his Figma, Marcelino had this as "Which platform would you like to preview" but that didn't sound right to me b/c `Dekstop` which is one of the choices is not a platform. Besides this questions is presented to the user when they don't provide a value for the `--device-type` flag (we don't have a `--platform` flag). So for these reasons. I reworded this but am open to changing it if needed.

@jshackell-sfdc what do you think?

Reviewer: ```suggestion
Which device type do you want to use for the preview?
```

Reviewer: yeah, let's stick with "device type", because that's what the name of the flag is, the help mentions, etc.

I made a slight editing suggestion. Cheers!
